release_notes.md

```diff
@@ -0,0 +1,67 @@
+ # Release notes
+
+- Woof Mobile APP
+- Release notes version: 1
```

Comment:
This number means only the order of the notes. 
I've deleted this information for not to cause misunderstanding.

Reply:
Would you mind, if  the file **release_notes.md** keeps only the title of the file and the app name, because we haven't made any releases yet, so , it could be empty. we've discussed this moment today with the team.

Reply:
Sure, just make sure the team knows the format of the file (what and how should be added when the file is being updated).